code-execution/src/test/java/com/dojo/codeexecution/service/GitManagerTest.java

```diff
@@ -149,4 +153,71 @@ public void buildParentWebHookWhenIsHookAvailable() throws IOException {
         verify(gitConfig, times(1)).getUser();
         verify(gitConfig, times(1)).getParentRepositoryName();
     }
+
+    @Nested
```

Comment:
Grouping methods this way and use more advanced techniques usually indicates bad architecture of the tested class.
Please think how we can separate and simplify.